### DISCUSSION TITLE
Typo in Controller props.

### DISCUSSION THREAD

**Discussion by fzkirablackwhy** [react-hook-form/react-hook-form#6718] [General] [answered]
https://github.com/react-hook-form/react-hook-form/blob/28368632adef25a8f1b20d17def340112c7f95e0/examples/V7/customMaskedInputWithController.tsx#L56

In version 7+ Controller has `render` property, not `as`  isn't it?

**Comment by bluebill1049** [comment]
oops, please send a PR if you can 🙏

**Reply by fzkirablackwhy** [reply]
sure!

**Reply by bluebill1049** [reply]
ty very much!

**Comment by fzkirablackwhy** [comment, marked as answer]
https://github.com/react-hook-form/react-hook-form/pull/6719/files

**Reply by bluebill1049** [reply]
thanks for sharing CSB as well. 🙏
